cmd/cni-metrics-helper/metrics/metrics.go

```diff
@@ -121,7 +117,7 @@ func processPercentile(metric *dto.Metric, act *metricsAction) {
 	act.actionFunc(&act.data.curSingleDataPoint, p99)
 }
 
-func processHistogram(metric *dto.Metric, act *metricsAction) {
+func processHistogram(metric *dto.Metric, act *metricsAction, log logger.Logger) {
```

Comment:
this is fine. might be a bit cleaner, though, to make this function a method on `metricsTarget` struct versus adding a `log` parameter to all these functions. :)

Reply:
I agree, but since `metricsTarget` is an interface, it will be a somewhat bigger change. Will check in a follow up PR how that would look like. 😄